webapp/commonComponents/validationTooltip.js

```diff
@@ -0,0 +1,24 @@
+import React from 'react'
+
+import * as ValidationUtils from '../utils/validationUtils'
+
+import * as R from 'ramda'
+
+import Tooltip from './tooltip'
+
+import useI18n from '../commonComponents/useI18n'
+
+export default ({ validation, children, className, type = '' }) => {
+  const i18n = useI18n()
+
+  const messagesHtml = ValidationUtils.getValidationFieldMessagesHTML(i18n)(validation)
+
+  return (
+    <Tooltip
+      type={type}
```

Comment:
Should ValidationTooltip always use ```type``` error?

Reply:
In theory yes, but the inner element (input and checkbox) get a different style when the tooltip class is tooltip-error - see tooltip.scss line 30. We could set an "error" class in the className instead?

Reply:
maybe ValiadationTooltip should conditionally pass the type param (error) to tooltip if validation is not valid?

Reply:
ok done